### PR TITLE
SBC Basics, Sensing, and Computer Vision

### PR DESCRIPTION
**Pull request by rfarber2328**
General information and links on SBC basics sensing and computer vision

### CODE REVIEW

**Comment by benbo** [comment]
These are pointers to very valuable resources and really good starting points for articles. But we want roboticsknowledgebase to become a primary resource for such information instead of a collection of links. Would it be possible to expand on these topics?

File: _data/navigation.yml [hunk 1]
```diff
@@ -68,6 +72,8 @@ wiki:
         url: /wiki/sensing/opencv-stereo/
       - title: Camera-IMU Calibration using kalibr
         url: /wiki/sensing/camera-imu-calibration/
+      - title: SBC Basics, Sensing, and Computer Vision
```
Comment (smbryan) [inline _data/navigation.yml]: "SBC" is a not a common acronym and obscures the article's intent. Recommend changing to "Single Board Computers for Computer Vision."

**Comment by smbryan** [comment]
The format also doesn't match what the site needs. Please take another look at [the template](https://github.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/blob/master/_templates/template.md)

File: wiki/fabrication/manufacturing-technique-selection-guide.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+Manufacturing Technique Selection Guide
```
Comment (smbryan) [inline wiki/fabrication/manufacturing-technique-selection-guide.md]: Should be:
`title: Manufacturing Technique Selection Guide`

File: wiki/fabrication/manufacturing-technique-selection-guide.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Laser Cutting Guide
+This link is to both an introductory and advanced topic laser cutting guide that seeks to answers all your laser cutting questions. The guide starts off by giving a brief history of lasers to the common laser cutting machines of today. Multiple diagrams explaining how a typical laser cutter works are provided that really help the reader understand what is physically going on when their design is cut. The guide also provides an in-depth set of guidelines for creating both laser cutter specific parts, like living hinges, as well as general rules for cutting basic mechanical devices like intermeshing gears. 
+
+## See Also:
```
Comment (smbryan) [inline wiki/fabrication/manufacturing-technique-selection-guide.md]: "See Also" is used specifically to help viewers find other content on the wiki and goes at the bottom of the page.

File: wiki/fabrication/manufacturing-technique-selection-guide.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- https://www.sculpteo.com/media/ebook/Sculpteo_ultimate_guide_laser_cutting.pdf
```
Comment (smbryan) [inline wiki/fabrication/manufacturing-technique-selection-guide.md]: Should be something like:
`[Sculpteo's Ultimate Guid to Laser Cutting](https://www.sculpteo.com/media/ebook/Sculpteo_ultimate_guide_laser_cutting.pdf)`

**Pull request closed without merging**